docs/infrastructure_configuration_variables.md

```diff
@@ -0,0 +1,37 @@
+# Infrastructure Configuration Variables
+# create a table
```

Comment:
I think this line (create a table) is not needed

Reply:
done